.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+on:
+  release:
+    types: [prereleased, released]
+
+jobs:
+  release:
+    name: ${{ github.event.release.tag_name }} to production
+    runs-on: ubuntu-latest
+    environment: production
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.release.tag_name }}
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          cache: npm
+      - name: Install dependencies
+        run: npm install
+      - name: Build
+        run: npm run build
+      - name: Deploy to server
+        uses: easingthemes/ssh-deploy@v4.1.8
+        with:
+          SSH_PRIVATE_KEY: ${{ secrets.SSH_KEY }}
+          REMOTE_HOST: ${{ secrets.HOST_IP }}
+          REMOTE_USER: ${{ secrets.HOST_USER }}
+          TARGET: /home/${{ secrets.HOST_USER }}/ace-server/
+          EXCLUDE: "/.cache/, /.github/, /.vscode/, /.git/, /node_modules/"
+          SCRIPT_AFTER: |
```

Comment:
`sudo pm2 reloaded` is need to reload the server. To test this, you can try to update atlassian-connect.json.